pallets/dapp-staking-v3/src/lib.rs

```diff
@@ -516,6 +516,12 @@ pub mod pallet {
     #[pallet::genesis_build]
     impl<T: Config> BuildGenesisConfig for GenesisConfig<T> {
         fn build(&self) {
+            if self == &Default::default() {
```

Comment:
@Dinonard a default test that runs GenesisConfig `build` with default GenesisConfig is added into substrate. I don't know what will be a default value for config so I thought as long as GenesisConfig is default/empty then don't run the build, hence avoid assert below

Reply:
I also commented it while doing review 🙂